akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/PersistentBehaviorSpec.scala

```diff
@@ -140,30 +140,30 @@ object PersistentBehaviorSpec {
         case IncrementTwiceAndThenLog ⇒
           Effect
             .persist(Incremented(1), Incremented(1))
-            .andThen {
+            .andThen { (_: State) ⇒
               loggingActor ! firstLogging
             }
-            .andThen {
+            .andThen { _ ⇒
               loggingActor ! secondLogging
             }
 
         case EmptyEventsListAndThenLog ⇒
           Effect
             .persist(List.empty) // send empty list of events
-            .andThen {
+            .andThen { _ ⇒
               loggingActor ! firstLogging
             }
 
         case DoNothingAndThenLog ⇒
           Effect
             .none
-            .andThen {
+            .andThen { _ ⇒
               loggingActor ! firstLogging
             }
 
         case LogThenStop ⇒
-          Effect.none
-            .andThen {
+          Effect.none[Event, State]
+            .andThen { _ ⇒
```

Comment:
These 2 examples, with `Effect.none.andThen`, look quite strange to me.

Can't/shouldn't we just do `case LogThenStop => { loggingActor ! firstLogging; Effect.stop }`?

Reply:
sure, but the purpose of these tests are to exercise the edge cases of chaining the callbacks, I presume.